frontend/public/i18n.js

```diff
@@ -53,6 +53,9 @@ i18n
         'console-app',
         'console-shared',
         'container-security',
+        'create-catalog-source',
```

Comment:
I hate to ask, but this PR is adding a lot of new namespaces. We're trying to consolidate. It would be better if we can move these to public before they are part of a translation drop. Hopefully this is just a find and replace, but it's difficult, let me know.

Reply:
done